cirq/google/calibration/workflow.py

```diff
@@ -501,7 +510,7 @@ def as_circuit(self) -> Circuit:
         return Circuit(self.operations)
 
 
-def run_floquet_characterization_for_circuit(
+def run_floquet_characterization_for_moments(
     circuit: Circuit,
```

Comment:
it's a little incongruous that this expects a `Circuit` but its name is "for moments"

Reply:
It is, I have a feeling this is not the last change of those names...